packages/webcomponents/src/components/filters/table-filters-menu.tsx

```diff
@@ -37,16 +37,16 @@ export class TableFiltersMenu {
     return (
       <div class="d-flex gap-2">
         <div class="dropdown pb-3">
-          <Button
-            variant="primary"
+          <button
```

Comment:
Changed this to generic html button? Was this a mistake? It used to be the UI component Button